devito/nodes.py

```diff
@@ -415,6 +415,20 @@ def name(self):
         return self._name
 
 
+class Denormals(Block):
+
+    """Macros to make sure denormal numbers are flushed in hardware."""
+
+    def __init__(self, header=None, body=None, footer=None):
+        b = [Element(c.Comment('Flush denormal numbers to zero in hardware')),
+             Element(c.Statement('_MM_SET_DENORMALS_ZERO_MODE(_MM_DENORMALS_ZERO_ON)')),
+             Element(c.Statement('_MM_SET_FLUSH_ZERO_MODE(_MM_FLUSH_ZERO_ON)'))]
```

Comment:
no, because apparently these need be re-hit any time a new parallel region is created (and threads re-created/re-initialised). Not sure why that is the case. Also, these need to be called by **each** thread, while in Python only the master thread would hit them